WalletWasabi.Fluent/Views/Wallets/Home/History/HistoryTable.axaml

```diff
@@ -13,6 +13,104 @@
+      <MenuItem Header="Copy Transaction ID"
+                Command="{Binding ClipboardCopyCommand}"
+                CommandParameter="{Binding Id, StringFormat={}{0}}">
+        <MenuItem.Icon>
+          <PathIcon Data="{StaticResource transaction_id}" />
+        </MenuItem.Icon>
+      </MenuItem>
+      <MenuItem Header="Copy Label"
+                Command="{Binding ClipboardCopyCommand}"
+                CommandParameter="{Binding Label, StringFormat={}{0}}">
+        <MenuItem.Icon>
+          <PathIcon Data="{StaticResource entities_regular}" />
+        </MenuItem.Icon>
+      </MenuItem>
+      <MenuItem Header="Copy Amount"
+                Command="{Binding  ClipboardCopyCommand}"
+                CommandParameter="{Binding IncomingAmount, StringFormat={}{0}}"
+                IsVisible="{Binding IncomingAmount, Converter={x:Static ObjectConverters.IsNotNull}}">
+        <MenuItem.Icon>
+          <PathIcon Data="{StaticResource btc_logo}" />
+        </MenuItem.Icon>
+      </MenuItem>
+      <MenuItem Header="Copy Amount"
+                Command="{Binding  ClipboardCopyCommand}"
+                CommandParameter="{Binding OutgoingAmount, StringFormat={}{0}}"
+                IsVisible="{Binding OutgoingAmount, Converter={x:Static ObjectConverters.IsNotNull}}">
+        <MenuItem.Icon>
+          <PathIcon Data="{StaticResource btc_logo}" />
+        </MenuItem.Icon>
+      </MenuItem>
+      <Separator />
+      <!-- TODO: Show speed up transaction dialog.
+      <MenuItem Header="Speed-up transaction"
+                IsVisible="{Binding !IsConfirmed}"
+                Command="{Binding SpeedUpTransactionCommand}">
+        <MenuItem.Icon>
+          <PathIcon Data="{StaticResource rocket_regular}" />
+        </MenuItem.Icon>
+      </MenuItem>
+      -->
+      <MenuItem Header="Show details"
+                Command="{Binding ShowDetailsCommand}">
+        <MenuItem.Icon>
+          <PathIcon Data="{StaticResource details_regular}" />
+        </MenuItem.Icon>
+      </MenuItem>
+    </ContextMenu>
+
+    <ContextMenu x:Key="CoinJoinContextMenu" x:DataType="historyItems:CoinJoinHistoryItemViewModel">
+      <MenuItem Header="Copy Amount"
+                Command="{Binding  ClipboardCopyCommand}"
+                CommandParameter="{Binding IncomingAmount, StringFormat={}{0}}"
+                IsVisible="{Binding IncomingAmount, Converter={x:Static ObjectConverters.IsNotNull}}">
+        <MenuItem.Icon>
+          <PathIcon Data="{StaticResource btc_logo}" />
+        </MenuItem.Icon>
+      </MenuItem>
+      <MenuItem Header="Copy Amount"
+                Command="{Binding  ClipboardCopyCommand}"
+                CommandParameter="{Binding OutgoingAmount, StringFormat={}{0}}"
+                IsVisible="{Binding OutgoingAmount, Converter={x:Static ObjectConverters.IsNotNull}}">
+        <MenuItem.Icon>
+          <PathIcon Data="{StaticResource btc_logo}" />
+        </MenuItem.Icon>
+      </MenuItem>
+      <Separator />
```

Comment:
`Copy Amount` should not exist because it copies the coinjoin fee.

Reply:
Fixed